### PR TITLE
Recorder popup titlebar.

### PR DESCRIPTION
**Pull request by csanyiarpad**
Creating the titlebar for the recorder popup:
- The popup gets the full screen.
- In the titlebar:
  - Blinkig red dot.
  - A button for closing it and going to the saving step.
  - A list of the recorder steps. You can remove any step you want,
    expect the very firts step with the open command, which is not
    shown in this list.

### CODE REVIEW

File: walkhub.css [hunk 1]
```diff
@@ -53,3 +53,110 @@
...
+
+.ui-dialog-titlebar.recording {
+  background-color: #eee;
+  border-bottom: 2px groove #999;
+  border-bottom-left-radius: 0px;
+  -moz-border-radius-bottomleft: 0px;
+  -webkit-border-bottom-left-radius: 0px;
+  border-bottom-right-radius: 0px;
+  -moz-border-radius-bottomright: 0px;
+  -webkit-border-bottom-right-radius: 0px;
```
Comment (Coornail) [inline walkhub.css:65]: Maybe it's time to convert to sass

File: walkhub.js [hunk 2]
```diff
@@ -15,7 +15,8 @@
     iframe: {
       name: "iFrame",
       linkcheck: false,
-      execute: function (url) {
+      execute: function (url, recording) {
+        recording = typeof recording !== "boolean" ? false : recording;
```
Comment (tamasd) [inline walkhub.js]: This is not necessary

File: walkhub.js [hunk 3]
```diff
@@ -407,6 +468,61 @@
     }, 1000);
   }
 
+  function resizeStepsDropdown(open) {
+    open = typeof open !== 'boolean' ? true : open;
```
Comment (tamasd) [inline walkhub.js]: This is not necessary

**Comment by Coornail** [comment]
Also pay attention to the Travis errors: https://travis-ci.org/Pronovix/WalkHub-module/builds/30828614

File: walkhub.css (hunk 1 above)
Comment (csanyiarpad) [inline walkhub.css:65]: Will do that later. I'm creating a ticket for it.

**Pull request merged**